python-sdk/tutorials/automl-with-azureml/ONNX_batch_model_generator_automl_for_images.py

```diff
@@ -0,0 +1,224 @@
+    parser.add_argument('--experiment_name', type=str, required=True,
+                        help='Name of the experiment')
+    parser.add_argument('--subscription_id', type=str, required=True,
+                        help='Subscription ID')
+    parser.add_argument('--resource_group', type=str, required=True,
+                        help='Resource group')
+    parser.add_argument('--workspace_name', type=str, required=True,
+                        help='Name of the workspace')
+    parser.add_argument('--run_id', type=str, required=True,
+                        help='id of the run')
+    parser.add_argument('--task_type', type=str, required=True,
+                        help='Task type in automl for images')
+    parser.add_argument('--box_score_thresh', type=float, required=True,
+                        help='During inference, only return proposals with a classification score \
+                        greater than box_score_thresh')
+
+    parser.add_argument('--min_size', type=int, required=False,
+                        help="Minimum size of the image to be rescaled before feeding it to the backbone")
+    parser.add_argument('--max_size', type=int, required=False,
+                        help="Maximum size of the image to be rescaled before feeding it to the backbone")
+    parser.add_argument('--box_nms_thresh', type=float, required=False,
+                        help='Non-maximum suppression (NMS) threshold for the prediction head')
+    parser.add_argument('--box_detections_per_img', type=int, required=False,
+                        help='Maximum number of detections per image, for all classes')
+    parser.add_argument('--img_size', type=int, required=False,
+                        help='Image size for train and validation')
+    parser.add_argument('--model_size', type=str, required=False,
+                        help='Size of the model from small, medium, large, xlarge')
+    parser.add_argument('--box_iou_thresh', type=float, required=False,
+                        help='IoU threshold')
+
+    # parse arguments
+    args = parser.parse_args()
+
+    ws = Workspace.create(name=args.workspace_name,
+                          subscription_id=args.subscription_id,
+                          resource_group=args.resource_group,
+                          exist_ok=True)
+    experiment = Experiment(ws, name=args.experiment_name)
+
+    # load the best child
+    automl_image_run = AutoMLRun(experiment=experiment, run_id=args.run_id)
+    best_child_run = automl_image_run.get_best_child()
+
+    model_type = None
+    if args.task_type == 'image-object-detection':
+        if args.model_name == 'yolo':
+            # yolo settings
+            model_settings = {"img_size": args.img_size, "model_size": args.model_size,
+                              "box_score_thresh": args.box_score_thresh, "box_iou_thresh": args.box_iou_thresh}
+        elif args.model_name == 'faster-rcnn':
```

Comment:
what happens for retinanet model?

Reply:
updated and tested it 
https://ml.azure.com/runs/rv-automl-image-object-detection-retinanet_1645044582_1c065fc9?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/rv-images-rc/workspaces/rv-images-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#outputsAndLogs